packages/ui/src/copiable-logo.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { cn } from "@dub/utils";
+import { Clipboard } from "lucide-react";
+import { MouseEvent, useCallback, useEffect, useState } from "react";
+import { createPortal } from "react-dom";
+import { toast } from "sonner";
+import { Button } from "./button";
+import { LogoType } from "./icons";
+
+const logoSvg = `<svg width="834" height="236" viewBox="0 0 834 236" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is there a better way to pull this logo SVG for copying it without waiting on a `fetch`?